docs/content/doc/usage/reverse-proxies.en-us.md

```diff
@@ -120,6 +120,14 @@ server {
 }
 ```
 
+## Resolving Error: 413 Request Entity Too Large
+
+This error indicates nginx is configured to restrict the file upload size.
+
+In your `nginx.conf`, find the `http { ... }` block and add the line
```

Comment:
This limit can be set per site and I would actually not recommend changing global setting

Reply:
It's allowed in contexts http, server, location. I guess for simple cases with one site it's preferred to set it on `http` level and we can't really cover all config situations.

Reply:
Yes but server should be used in most cases imho in sites-available.d/default conf file

Reply:
I never use those site files on single-site setups, but yeah I guess you have a point as that site-specifc config is the recommended variant.

Reply:
```suggestion
In your nginx config file containing your Gitea proxy directive, find the `location { ... }` block for Gitea and add the line
```